packages/editor/src/components/sidebar/index.js

```diff
@@ -28,7 +29,6 @@ import PluginSidebar from '../plugin-sidebar';
 import PostSummary from './post-summary';
 import PostTaxonomiesPanel from '../post-taxonomies/panel';
 import PostTransformPanel from '../post-transform-panel';
-import SidebarHeader from './header';
```

Comment:
We can delete this file

Reply:
Done in [2d23a17](https://github.com/WordPress/gutenberg/pull/63184/commits/2d23a17d1eb3391a0b2c508ef9bb4a15c7c4bbcc)